examples/04_example_patchextraction.ipynb

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
<p>You are not running the latest version of tiatoolbox to run this notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TIA-Lab/tiatoolbox/pull/147/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Checked with latest version 0.7.0</p>